### PR TITLE
Squeeze implementation

### PR DESCRIPTION
**Pull request by spidyDev**
Added "Squeeze" operator support

1. "Squeeze" operator implementation.
2.  Changed backend.py to support "squeeze" test.
3.  Added unit test in "test_layers.py"

@Roshrini @lupesko @anirudhacharya @nswamy

### CODE REVIEW

File: onnx_mxnet/import_onnx.py [hunk 1]
```diff
@@ -135,9 +135,13 @@ def from_onnx(self, graph):
             elif op_name == 'AveragePool' and onnx_attr.get('pads') is not None or \
                                     op_name == 'MaxPool' and onnx_attr.get('pads') is not None:
                 op = self._fix_pooling(op_name, inputs, onnx_attr)
+            elif op_name == 'Squeeze':
+                op = self._fix_squeeze(inputs, mx_attr)
             else:
                 op = new_op(name=node_name, *inputs, **mx_attr)
 
+
```
Comment (Roshrini) [inline onnx_mxnet/import_onnx.py]: nitpick: extra lines?

File: onnx_mxnet/tests/test_layers.py [hunk 2]
```diff
@@ -216,5 +216,14 @@ def test_softmax(self):
         numpy_op = exp_score / exp_score.sum(0)
         npt.assert_almost_equal(output, numpy_op)
 
+    def test_squeeze(self):
+        """Test for squeeze operator"""
+        node_def = helper.make_node("Squeeze", ["ip1"], ["op1"], axes=[1, 3])
+        ip1_shape = (3, 1, 2, 1, 4)
+        ip1 = np.random.ranf(ip1_shape).astype("float32")
```
Comment (Roshrini) [inline onnx_mxnet/tests/test_layers.py]: Call _random_array() function, which is defined in the same file, passing "ip1_shape" as input to do the same.

File: onnx_mxnet/import_onnx.py [hunk 3]
```diff
@@ -206,6 +212,20 @@ def _fix_slice(self, inputs, new_attr):
                 slice_op = mx.sym.slice_axis(slice_op, axis=axis, begin=begin[i], end=end[i])
         return slice_op
 
+    def _fix_squeeze(self, inputs, new_attr):
```
Comment (Roshrini) [inline onnx_mxnet/import_onnx.py]: Should we create an issue in MXNet to add support for squeeze operator and add a link to the issue here in comment?

Comment (spidyDev) [reply onnx_mxnet/import_onnx.py]: MXNet added it last week , but is not part of current release plan. Once it gets in release we can change this functionality.

File: onnx_mxnet/import_onnx.py [hunk 4]
```diff
@@ -206,6 +210,20 @@ def _fix_slice(self, inputs, new_attr):
...
+        """
+        MXNet doesnt have a squeeze operator.
+        Using "split" to perform similar operation.
+        "split" can be slower compared to "reshape".
+        Remove this implementation once mxnet adds the support.
+        """
+        axes = new_attr.get('axis')
+        op = mx.sym.split(inputs[0], axis=axes[0], num_outputs=1, squeeze_axis=1)
+        if len(axes) > 1:
```
Comment (nswamy) [inline onnx_mxnet/import_onnx.py]: nit: this is redundant, for will not execute if < 2

**Pull request merged**